service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -0,0 +1,455 @@
+
+.RbacConfig, ServiceRole, ServiceRoleBinding (rbac.istio.io/v1alpha1)
+
+These resources have been replaced by the `security.istio.io/v1beta1` AuthorizationPolicy resource.
+
+Mimicking RbacConfig behavior requires writing a default AuthorizationPolicy whose settings depend on the spec.mode specified in the RbacConfig.
+
+* When `spec.mode` is set to `OFF`, no resource is required as the default policy is ALLOW, unless an AuthorizationPolicy applies to the request.
+* When `spec.mode` is set to ON, set `spec: {}`.  You must create AuthorizationPolicy policies for all services in the mesh.
+* `spec.mode` is set to `ON_WITH_INCLUSION`, must create an AuthorizationPolicy with `spec: {}` in each included namespace. Inclusion of individual services is not supported by AuthorizationPolicy. However, as soon as any AuthorizationPolicy is created that applies to the workloads for the service, all other requests not explicitly allowed will be denied.
+* When `spec.mode` is set to `ON_WITH_EXCLUSION`, it is not supported by AuthorizationPolicy. A global DENY policy can be created, but an AuthorizationPolicy must be created for every workload in the mesh because there is no allow-all policy that can be applied to either a namespace or a workload.
+
+AuthorizationPolicy includes configuration for both the selector to which the configuration applies, which is similar to the function ServiceRoleBinding provides and the rules which should be applied, which is similar to the function ServiceRole provides.
+
+.ServiceMeshRbacConfig (maistra.io/v1)
+
+This resource is replaced by using a `security.istio.io/v1beta1` AuthorizationPolicy resource with an empty spec.selector in the control plane’s namespace.  This policy will be the default authorization policy applied to all workloads in the mesh.  For specific migration details, see RbacConfig above.
+
+[id="ossm-upgrading-mig-mixer_{context}"]
+=== Mixer plugins
+
+Mixer components are disabled by default in version 2.0. If you rely on Mixer plugins for your workload, you must configure your version 2.0 `ServiceMeshControlPlane` to include the Mixer components. 
+
+To enable the Mixer policy components, add the following snippet to your `ServiceMeshControlPlane`. 
+
+[source,yaml]
+----
+spec:
+  policy:
+    type: Mixer
+----
+
+To enable the Mixer telemetry components, add the following snippet to your `ServiceMeshControlPlane`. 
+
+[source,yaml]
+----
+spec:
+  telemetry:
+    type: Mixer
+----
+
+Legacy mixer plugins can also be migrated to WASM and integrated using the new ServiceMeshExtension (maistra.io/v1alpha1) custom resource.
+
+WASM filters from upstream Istio community are not supported in {ProductName} 2.0.
```

Comment:
I think we should clarify this to be, "Built-in WASM filters included in the upstream Istio distribution are not available in {ProductName} 2.0."